docs/modules/kubernetes/pages/helm-upgrade-guide.adoc

```diff
@@ -0,0 +1,140 @@
+
+We recommend to disable Management Center before upgrade, and re-enable it after upgrade. You should follow the steps outlined below:
+
+Step 1: Export values of existing installation:
+
+[source,bash]
+----
+helm get values <installation-name> > values.yaml
+----
+
+Step 2: Disable Management Center:
+
+[source,bash]
+----
+helm upgrade <installation-name> hazelcast/hazelcast-enterprise -f values.yaml --version <installed-version> --set mancenter.enabled=false
+----
+
+Step 3: Delete persistent volume of the deleted Management Center:
+
+[source,bash]
+----
+kubectl get pvc -o name | grep "mancenter" | xargs -n 1 kubectl delete
+----
+
+Step 4: Upgrade your installation without disabling Management Center:
+
+[source,bash]
+----
+helm upgrade hz hazelcast/hazelcast-enterprise -f values.yaml --version <new-version>
+----
+
+
+The following solutions upgrade solutions cover scenarios for persistence/hotRestart enabled installations:
+
+== Upgrading from Hazelcast Version 4.x to 5.x
+
+Hazelcast rolling update supports upgrading from 4.2 to 5.0. You can check link:https://docs.hazelcast.com/hazelcast/5.3/maintain-cluster/rolling-upgrades#hazelcast-members-compatibility-guarantees[Hazelcast's Compatibility Guarantees] for more information.
+
+=== Upgrading from Helm Chart < 3.11 Installations
+
+These versions do not support rolling upgrade, so we propose you to check Alternative Solutions section.
+
+=== Upgrading from Helm Chart 3.11 Installations
+
+3.11 and 5.3.9 are recent versions that upgrade problems are fixed, so that they are compatible. If you are using 3.11, we recommend upgrading to 5.3.9:
+
+[source,bash]
+----
+helm get values > values.yaml
+helm upgrade hz hazelcast/hazelcast-enterprise --version 5.3.9 -f values.yaml
+----
+
+== Upgrading from Hazelcast Version 5.x to 5.x
+
+We support Hazelcast rolling upgrade and chart upgrade from 5.4.7 to 5.9.2. However, in 5.8.0, there is a breaking change: `hot-restart-persistence` is renamed to `persistence`. Following steps should be applied:
+
+Step 1: Export values of existing installation
+
+[source,bash]
+----
+helm get values <installation-name> > values.yaml
+----
+
+Step 2: Modify existing values
+
+- Optional: `hot-restart-persistence` should be replaced with `persistence` in Hazelcast configuration. See: link:https://docs.hazelcast.com/hazelcast/5.2/storage/configuring-persistence[Configuring Persistence]. Old configuration is still supported in newer Hazelcast versions, so it is an optional step.
+
+- `hotRestart` field in `values.yaml` should be replaced with `persistence`. See: link:https://docs.hazelcast.com/hazelcast/5.3/kubernetes/helm-hazelcast-enterprise-chart#5-8-0[Notable Changes in Helm Chart]
+
+Step 3: Upgrade using corrected values
+
+[source,bash]
+----
+helm upgrade hz hazelcast/hazelcast-enterprise --version 5.8.0 -f values.yaml
+----
+
+If you are using versions other than in these ranges, please check the next section.
+
+== Alternative Solutions
+
```

Comment:
The following alternative methods can be used:

* xref:helm-upgrade-guide.adoc#persistence-based[]
* xref:helm-upgrade-guide.adoc#retain-helm-chart-version[]

Reply:
I think this is part of the Upgrade with Persistence Enabled section, if so, the heading needs to be a level lower (=== Alternative instead of == Alternative)